### PR TITLE
Enahanced the Heat harvesting tool in the Visual Studio extension to prov...

### PR DESCRIPTION
**Pull request by blamoureux**
...ide support for harvesting refernced DLLs in CSPROJ files

### CODE REVIEW

File: src/ext/VSExtension/wixext/VSHeatExtension.cs [hunk 1]
```diff
@@ -11,6 +11,9 @@
 // </summary>
 //-------------------------------------------------------------------------------------------------
 
+using System.Net.Mime;
+using System.Text.RegularExpressions;
+
```
Comment (icpenguins) [inline src/ext/VSExtension/wixext/VSHeatExtension.cs:16]: Typically, as seen below, a using goes inside the namespace. Also, what is the purpose of System.Net.Mime here?

File: src/ext/VSExtension/wixext/VSHeatExtension.cs [hunk 2]
```diff
@@ -153,6 +156,15 @@ public override void ParseOptions(string type, string[] args)
                         bool found = false;
                         foreach (string availableOutputGroup in allOutputGroups)
                         {
+                            // Allow for specifying the name along with a Metadata filter that can be applied to include/exclude items
+                            // The syntax for that will be "{pogName}:{Metadata Filter Expression}"
+                            // eg.  References:CopyLocal=true   or References:CopyLocal!=true
+                            Match expressive = Regex.Match(pogName, "([^:]+):(.*)", RegexOptions.IgnoreCase);
```
Comment (icpenguins) [inline src/ext/VSExtension/wixext/VSHeatExtension.cs:162]: Do you need to use Regex here? It seems like you are just looking for ':' and want to split on that?

File: src/ext/VSExtension/wixext/VSHeatExtension.cs (hunk 1 above)
Comment (blamoureux) [inline src/ext/VSExtension/wixext/VSHeatExtension.cs:16]: Thanks Brian,

I will revise.
ReSharper jammed that Net.Mime in there.  I didn't notice the pollutuion.
I will correct this

regarding the Regex, I wanted to handle the case where there can be
semi-colons in the metadata filter.  For example, filtering on a HintPath
of C:\bin  could be used.
The Regex finds the first occurance.  Could certainly consider using .First
instead

**Comment by robmen** [comment]
Thank you but two things are required to move forward with this pull request:
1. Assignment agreement needs to be signed: http://wixtoolset.org/development/assignment-agreement/
2. Code should be updated to better fit with the surrounding code, namely remove usage of `var`, `LINQ` query syntax and general whitespace.

I'm going to close this pull request to clear out the back log. If the two issues can be resolved, please do feel free to reopen the pull request to the updated branch.

**Pull request closed without merging**